deployments/kubernetes/chart/reloader/values.yaml

```
@@ -78,6 +78,7 @@ reloader:
     resources: {}
     pod:
       annotations: {}
+    priorityClassName: "foo"
```

```suggestion
    # priorityClassName: "foo"
```

Please keep this commented out since we don't want this as a default value.

---

Sorry must have missed this after testing.